modules/graphql_content/src/Plugin/Deriver/RawValueFormatterDeriver.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Drupal\graphql_content\Plugin\Deriver;
+
+use Drupal\Component\Plugin\Derivative\DeriverBase;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\Plugin\Discovery\ContainerDeriverInterface;
+use Drupal\Core\Entity\EntityFieldManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Generate a FieldFormatter plugin applicable to all available field types.
+ */
+class RawValueFormatterDeriver extends DeriverBase implements ContainerDeriverInterface {
+
+  /**
+   * The entity field manager.
+   *
+   * @var \Drupal\Core\Entity\EntityFieldManagerInterface
+   */
+  protected $entityFieldManager;
+
+  /**
+   * The base plugin id.
+   *
+   * @var string
+   */
+  protected $basePluginId;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, $basePluginId) {
+    return new static(
+      $container->get('entity_field.manager'),
+      $basePluginId);
+  }
+
+  /**
+   * AbstractFieldFormatterDeriver constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityFieldManagerInterface $entityFieldManager
+   *   An entity field manager instance.
+   * @param string $basePluginId
+   *   The base plugin id.
+   */
+  public function __construct(
+    EntityFieldManagerInterface $entityFieldManager,
+    $basePluginId
+  ) {
+    $this->basePluginId = $basePluginId;
+    $this->entityFieldManager = $entityFieldManager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getDerivativeDefinitions($basePluginDefinition) {
+    $fieldTypes = [];
+
+    foreach ($this->entityFieldManager->getFieldMap() as $fields) {
+      foreach ($fields as $fieldName => $info) {
+        $fieldTypes[] = $info['type'];
+      }
+    }
+
+    $this->derivatives = [
```

Comment:
Isn't $this->derivatives normally a keyed list of derivatives? In this case you made it $derivatives[0] = [...]